phase1/openstack/README.md

```diff
@@ -0,0 +1,44 @@
+# Getting started on OpenStack
+
+Prerequisites:
+* docker-engine which can be downloaded [here](https://docs.docker.com/engine/installation/)
```

Comment:
We should also add that a working OpenStack install with: Keystone v3, Nova and Neutron (with LBaaS v2) is required.